src/Voron/Data/BTrees/Tree.cs

```diff
@@ -126,6 +121,11 @@ public bool IsLeafCompressionSupported
 
         public bool HasNewPageAllocator { get; private set; }
 
+        public static Tree GetRoot(LowLevelTransaction llt, Slice name, TreeMutableState parentState)
```

Comment:
Given that this is root, does it make sense to have `name` parameter? Shouldn't it always be `Constants.RootTreeNameSlice`?